src/delagent/ui/admin-upload-delete.php

```diff
@@ -114,27 +115,79 @@ function Delete($uploadpk, $Depends = NULL)
   } // Delete()
 
   /**
+   * @param $uploadpks
+   * @brief starts deletion and handles error messages
+   * @return string
+   */
+  function initDeletion($uploadpks)
+  {
+    $V = "";
+    $errorMessages = [];
+    $deleteResponse = "";
+    for($i=0; $i < sizeof($uploadpks); $i++)
+    {
+      $deleteResponse = $this->TryToDelete($uploadpks[$i]);
+
+      if($deleteResponse->getDeleteMessageCode() != DeleteMessages::SUCCESS)
+      {
+        $errorMessages[] = $deleteResponse;
+      }
+    }
+
+    /**
+     * Output normal message, if only one upload
+     */
+    if(sizeof($uploadpks) < 2)
+    {
+      $V .= DisplayMessage($deleteResponse->getDeleteMessageString().$deleteResponse->getAdditionalMessage());
+    }
+    else
+    {
+      $displayMessage = "";
+
+      if(in_array(DeleteMessages::SCHEDULING_FAILED, $errorMessages))
+      {
+        $displayMessage .= "<br/>Scheduling failed for " .
```

Comment:
The breaks are alright in this case from a visual standpoint. It's done so that it only breaks the warning messages., not the info message. That's why you got breaks at the end of the other cases